### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by RubenKelevra**

### CODE REVIEW

**Review by MrBartusek — COMMENTED** [review]
Hey, thanks for the pr! German translation still misses `common.unavailable` keys, could you update them too?

File: src/translations/de.json [hunk 1]
```diff
@@ -4,55 +4,55 @@
...
         },
         "red": {
-          "event": "{event} der Warnstufe Rot",
-          "generic": null,
-          "color": null
+          "event": "Rote Warnung: {event}",
+          "generic": "Rote Warnung",
+          "color": "Rot"
         }
     }
```
Comment (MrBartusek) [inline src/translations/de.json:57]: `event` keys should feel like real sentences, you can see example in [english translation](https://github.com/MrBartusek/MeteoalarmCard/blob/master/src/translations/en.json#L41-L56)

**Comment by MrBartusek** [comment]
Hey @jpbede @nurtext mind taking a look at this PR since you are German translation contributors?

**Comment by nurtext** [comment]
@MrBartusek Looks good to me.

File: src/translations/de.json (hunk 1 above)
Comment (RubenKelevra) [reply src/translations/de.json:57]: Yeah, but the option to do this better (than before) still sounds weird in German. This is the best solution at the available length - and sounds pretty much like spoken German.

**Comment by jpbede** [comment]
@MrBartusek LGTM 👍

**Review by MrBartusek — APPROVED** [review]

**Comment by MrBartusek** [comment]
Thank you all!

**Pull request merged**